AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginAuthIntegrationTests/MultiAuth/README.md

```diff
@@ -0,0 +1,68 @@
+## DataStore with Multi-Auth Integration Tests
+
+### Prerequisites
+- AWS CLI
+
+### Set-up
+
+1. create a `schema.graphql` file with the content of  `MultiAuth/schema.graphql`
+
+2. `amplify init`
+
+3. `amplify add api`
+```
+? Please select from one of the below mentioned services: GraphQL
+? Provide API name: datastoreintegtestmu
+? Choose the default authorization type for the API API key
+? Enter a description for the API key:
+? After how many days from now the API key should expire (1-365): 365
+? Do you want to configure advanced settings for the GraphQL API Yes, I want to make some additional changes.
+? Configure additional auth types? Yes
+? Choose the additional authorization types you want to configure for the API: Amazon Cognito User Pool, IAM, OpenID Connect
+Cognito UserPool configuration:
+Use a Cognito user pool configured as a part of this project.
+OpenID Connect configuration:
+? Enter a name for the OpenID Connect provider: okta
+? Enter the OpenID Connect provider domain (Issuer URL): [OIDC_URL_HERE]
+? Enter the Client Id from your OpenID Client Connect application (optional):
+? Enter the number of milliseconds a token is valid after being issued to a user: 86400000
+? Enter the number of milliseconds a token is valid after being authenticated: 86400000
+? Enable conflict detection? Yes
+? Select the default resolution strategy Auto Merge
+? Do you have an annotated GraphQL schema? Yes
+? Provide your schema file path: schema.graphql
+```
+4. `amplify push`
+```perl
+? Are you sure you want to continue? `Yes`
+? Do you want to generate code for your newly created GraphQL API `No`
+```
+
+5. Copy `amplifyconfiguration.json` to a new file named `AWSDataStoreCategoryPluginMultiAuthIntegrationTests-amplifyconfiguration.json`
+6. Create `AWSDataStoreCategoryPluginMultiAuthIntegrationTests-credentials.json` with the following value
+```json
+{
+    "user1": "<USER1_EMAIL>",
```

Comment:
might be easier to just use a username instead of email, so if you update this, should also update the steps in `amplify add api` -> User pool DefaultConfiguration -> sign in with username